startup.d/20-syntax-check.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+python -m py_compile $(find . -name '*.py');
+if [ "$?" != "0" ]
+then
+  echo "There is syntax error. This may endanger chaos. Pauses updates."
+  killall bash
```

Comment:
"in-memory" where? Chaos' process is overwritten so that's out. It's not feasible for Supervisor to have a backup copy of the entire process and it wouldn't work even if it did. Does Supervisor somehow magically back up all the files needed to run Chaos when it first runs Chaos? Point me to the documentation which says that. If there's some other "in-memory" place that I'm missing, explain it in detail.